src/OsiCbcSolverInterface.cpp

```diff
@@ -7,6 +7,8 @@
 #include "OsiCbcSolverInterface_2_8.cpp"
```

Comment:
I think we decided to include the `hpp` file here instead of duplicating this block